Detectors/TRD/base/src/PadPlane.cxx

```diff
@@ -175,7 +175,7 @@ double PadPlane::getPad(double y, double z) const
     tiltOffsetPad = tiltOffsetY * mInverseWidthIPad;
     pad += tiltOffsetPad;
   }
-
+  LOGP(info, " pad : {} ", pad);
   assert(!(pad < 0.0 || pad > double(mNcols)));
```

Comment:
why not just remove the assert and force it to the boundaries? In any case this function seems to be used only in the test macro which fails and causes CI problems now

Reply:
just waiting for a mac arm to finish building, my concern is the error is somewhere else and this is just the manifestation of it. Yes, the plan was if its not obvious will just pad to the which ever boundary its closest to

Reply:
assert changed to if statements, and will investigate properly when my arm build of O2 finishes.